components/src/components/SimpleBCAddress/Component.ts

```diff
@@ -16,83 +15,83 @@ const ID = 'simplebcaddress';
 const DISPLAY = 'Simple BC Address';
 
 export default class Component extends (ParentComponent as any) {
-    static schema(...extend) {
-
-        return ParentComponent.schema({
-            label: DISPLAY,
-            type: ID,
-            key: ID,
-            provider: 'custom',
-            providerOptions: {
-                queryProperty: 'addressString',
-              url:import.meta.env.VITE_CHEFS_ADVANCE_GEO_ADDRESS_APIURL},
-
-            queryParameters:{"echo": false,
-            "brief": true,
-            "minScore": 55,
-            "onlyCivic": true,
-            "maxResults": 15,
-            "autocomplete": true,
-            "matchAccuracy": 100,
-            "matchPrecision": "occupant, unit, site, civic_number, intersection, block, street, locality, province",
-            "precisionPoints": 100,}
-
-        }, ...extend);
-    }
-
-    static get builderInfo() {
-        return {
-            title: DISPLAY,
-            group: 'advanced',
-            icon: 'address-book',
-            weight: 90,
-            documentation: Constants.DEFAULT_HELP_LINK,
-            schema: Component.schema(),
-
-        };
-    }
-
-    public static editForm = editForm;
-
-    async init(){
-        super.init();
-    }
-
-    async attach(element) {
-        super.attach(element);
-        try {
-
-            let {
-                providerOptions,
-                queryParameters,
-
-            } = this.component;
-
-            if(providerOptions) {
-                if(!providerOptions.params) {
-                    providerOptions["params"]={}
-                }
-                if(queryParameters) {
-                    providerOptions.params ={...providerOptions.params,...queryParameters}
-                }
-            }
-
-        } catch (err) {
-            console.log(`This error is from Custom BC Address component in form.io: Failed to acquire configuration: ${err.message}`);
+  static schema(...extend) {
+    return ParentComponent.schema(
+      {
+        label: DISPLAY,
+        type: ID,
+        key: ID,
+        provider: 'custom',
+        providerOptions: {
+          queryProperty: 'addressString',
+          url: import.meta.env.VITE_CHEFS_GEO_ADDRESS_APIURL,
```

Comment:
This is the only code change in the file - use `VITE_CHEFS_GEO_ADDRESS_APIURL` instead of `VITE_CHEFS_ADVANCE_GEO_ADDRESS_APIURL`.